### ISSUE TITLE
Useless parenthesis logical and

### ISSUE THREAD

**Issue by marcospassos** [slevomat/coding-standard#634]
The sniff reports useless parenthesis for the following statement:

```php
$hash = (31 * $hash ) & 0xFFFFFFFF;
```
Should it be allowed as it is much more clear?

**Comment by kukulich** [comment]
The result is same with or without the parentheses so the parentheses are useless.
